docs/dev/tools/common-errors.md

```diff
@@ -233,8 +233,48 @@ If not, look for other related errors in your logs.
 
 **Solution:**
 
-1. Verify your machine's internet connection is stable and working properly.
-1. If you are using API keys to connect, ensure you are using the correct credentials for the machine.
+1. Check networking:
+
+   1. Check if your machine is showing as online on Viam.
+   1. Verify your machine's internet connection is stable and working properly.
+   1. Ensure network checks on `viam-server` succeed by using the Go SDK's [`client.WithNetworkStats()`](https://pkg.go.dev/go.viam.com/rdk/robot/client#WithNetworkStats) method when connecting:
```

Comment:
this should be in the check sdk connection section. The viam-server runs network checks automatically after start up, so tell users to look at the logs maybe?